azure-pipelines.yaml

```diff
@@ -60,7 +63,10 @@ parameters:
 
 variables:
   - name: tfversion
-    value: 0.15.1
+    ${{ if ne(parameters['components.deployment'], 'shutter') }}:
+      value: 0.15.1
```

Comment:
why can't we upgrade this for all?

Reply:
could be done, wasn't sure if we wanted to keep the existing infra as is